pkg/snapshot/v1alpha1/snapshot_template_engine.go

```diff
@@ -34,6 +34,26 @@ import (
 type snapshotEngine struct {
 	// engine exposes generic CAS template operations
 	engine engine.Interface
+	// defaultConfig is the default cas volume configurations found
```

Comment:
@princerachit Can you put a TODO to move these stuff to core cast engine?
Similar logic is repeated in VolumeEngine. It will be good to make it part of the core.